README.adoc

```diff
@@ -239,8 +238,8 @@ Email = libbybot@openliberty.io
 SerialNumber = Liberty0001
 ----
 
-After clicking the `Submit` button, you will not see any changes since there are no constraint
-violations. Now try the following values, all of which break the previously defined constraints:
+After you click the `Submit` button, you won't see any changes because constraint
```

Comment:
The issue here is "there are." Maybe instead say, "After you click the `Submit` button, you won't see any changes because constraint violations do not exist." or "...because contraint violations are not produced."?